modules/getting-started/proc-techdocs-configure-odf-helm.adoc

```diff
@@ -0,0 +1,34 @@
+[id="proc-techdocs-configure-odf-helm_{context}"]
+
+= Making object storage accessible to containers using the Helm chart
+
+`ObjectBucket` information is stored in the {product-short} `ObjectBucketClaim` config map and secret. Adding the `ObjectBucket` information to the Helm chart configuration makes it accessible to the {product-short} container by adding the following environment variables to the container:
+
+* `BUCKET_NAME`
+* `BUCKET_HOST`
+* `BUCKET_PORT`
+* `BUCKET_REGION`
+* `BUCKET_SUBREGION`
+* `AWS_ACCESS_KEY_ID`
+* `AWS_SECRET_ACCESS_KEY`
+
+These variables are then used in the TechDocs plugin configuration.
+
+.Prerequisites
+
+* You have installed {product} on {ocp-short} using the Helm chart.
+* You have created an `ObjectBucketClaim` CR for storing files generated by TechDocs.
+
+.Procedure
+
+* In the `upstream.backstage` key in the Helm chart values, enter the name of the Developer Hub `ObjectBucketClaim` secret as the value for the `extraEnvVarsSecrets` field and the `extraEnvVarsCM` field. For example:
++
+[source]
```

Comment:
```suggestion
[source,yaml]
```
The reason for using `[source]` on the codeblocks is so that we can use the syntax highlighting by specifying which language the code block is in. If we're not specifying the languages, you don't actually need to include these, but I think it's nice to have the right language added.